central/graphql/resolvers/node_vulnerabilities.go

```diff
@@ -80,8 +143,243 @@ func (resolver *Resolver) NodeVulnCounter(ctx context.Context, args RawQuery) (*
 		query := withNodeCveTypeFiltering(args.String())
 		return resolver.vulnCounterV2(ctx, RawQuery{Query: &query})
 	}
-	// TODO : Add postgres support
-	return nil, errors.New("Resolver NodeVulnCounter does not support postgres yet")
+
+	if err := readNodes(ctx); err != nil {
+		return nil, err
+	}
+	query, err := args.AsV1QueryOrEmpty(search.ExcludeFieldLabel(search.Fixable))
+	if err != nil {
+		return nil, err
+	}
+	vulnLoader, err := loaders.GetNodeCVELoader(ctx)
+	if err != nil {
+		return nil, err
+	}
+	query = tryUnsuppressedQuery(query)
+	fixableVulnsQuery := search.ConjunctionQuery(query, search.NewQueryBuilder().AddBools(search.Fixable, true).ProtoQuery())
+	fixableVulns, err := vulnLoader.FromQuery(ctx, fixableVulnsQuery)
+	if err != nil {
+		return nil, err
+	}
+
+	unFixableVulnsQuery := search.ConjunctionQuery(query, search.NewQueryBuilder().AddBools(search.Fixable, false).ProtoQuery())
+	unFixableCVEs, err := vulnLoader.FromQuery(ctx, unFixableVulnsQuery)
+	if err != nil {
+		return nil, err
+	}
+
+	return mapNodeCVEsToVulnerabilityCounter(fixableVulns, unFixableCVEs), nil
```

Comment:
Check out the new map function and code flows as added in #2253

Reply:
Updated the code flow